multiversx_sdk_cli/tests/test_cli_staking_provider.py

```diff
@@ -15,11 +15,11 @@ def test_create_new_delegation_contract(capsys: Any):
     main([
         "staking-provider", "create-new-delegation-contract",
         "--pem", str(alice),
-        "--recall-nonce", "--estimate-gas",
+        "--nonce", "7", "--estimate-gas",
         "--value", "1250000000000000000000",
         "--total-delegation-cap", "10000000000000000000000",
         "--service-fee", "100",
-        "--proxy", "https://testnet-api.multiversx.com"
+        "--chain", "T"
```

Comment:
Proxy not needed here?

Reply:
No, because we are not sending the transaction. We used the proxy just to get the nonce which in our test it's not really relevant.